fluentui_topappbars/src/main/java/com/microsoft/fluentui/tokenized/AppBar.kt

```diff
@@ -179,6 +197,11 @@ fun AppBar(
                                         postTitleIcon.value(themeMode),
                                         postTitleIcon.contentDescription,
                                         modifier = Modifier
+                                            .then(
```

Comment:
Icon should flip instead of rotating. As the rotate won't work on asymmetric icon.
Take input in FluentIcon data class, if the icon need to be flipped in RTL. As there are icon which should not be flipped at all in RTL or need to be provided an explicit icon for RTL scenario.